### PR TITLE
Add a sample how to load an extension language file

### LINKED ISSUE #25: language/usage.rst should have a short section about loading extension files
with `add_lang_ext()`

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #25

### CODE REVIEW

File: development/language/usage.rst [hunk 1]
```diff
@@ -8,48 +8,96 @@ also a requirement for Extensions to use said system for all output presented to
...
+To load a file from an extension
+you need to use ``phpbb\user::add_lang_ext()`` which takes
+the vendor + extension name as first argument and the array of language files as
+a second argument.
 
 .. code-block:: php
 
-    $message .= '<br /><br />' . $user->lang('MY_TRICKY_LANGUAGE_KEY', '<a href="' . append_sid("{$phpbb_root_path}mypage.$phpEx") . '">', '</a>');
-    trigger_error($message);
+    $user->add_lang_ext('acme/demo', array('demo'));
```
Comment (nickvergessen) [inline development/language/usage.rst]: should also explain single load (without array)

File: development/language/usage.rst [hunk 2]
```diff
@@ -57,7 +105,9 @@ Using the Language System in template files
 Language entries in
 `Using the Template System <https://wiki.phpbb.com/Using_the_phpBB3.0_Template_System>`_
 are a major improvement since phpBB 3.0. There is no longer a need to manually
-assign these in the php file; loaded language files will be automatically used.
+assign these in the PHP file; language entries of loaded language files can be
+used automatically.
+
 To use the language entry with the key ``MY_KEY`` in a template file, just write
 ``{L_MY_KEY}`` in the template - it's as simple as that.
```
Comment (nickvergessen) [inline development/language/usage.rst]: Remove ` - it's as simple as that`

**Comment by nickvergessen** [comment]
@CHItA mind reviewing, so we can afterwards for the master docs correctly?
That would be awesome :smiley:

**Comment by CHItA** [comment]
It looks good for me.

**Pull request merged**